app/views/items/_form.html.erb

```diff
@@ -1,7 +1,12 @@
 <%= simple_form_for @item, :html => {:multipart => true} do |f| %>
+
   <%= f.error_notification %>
 
   <div class="form-inputs">
+    <%= f.association :vendor, label: "Vendor ", collection: Vendor.order("name") %>
```

Comment:
This should not be the very first thing on the form. Probably leave it in the same position `Source` was in previously ?